checkov/main.py

```diff
@@ -757,15 +762,20 @@ def run(self, banner: str = checkov_banner) -> int | None:
         bc_integration.get_prisma_build_policies(self.config.policy_metadata_filter)
 
         integration_feature_registry.run_pre_scan()
-
+        policy_level_suppression = suppressions_integration.get_policy_level_suppressions()
+        bc_cloned_checks = custom_policies_integration.bc_cloned_checks
+        runner_filter.bc_cloned_checks = bc_cloned_checks
+        custom_policies_integration.policy_level_suppression = policy_level_suppression
         runner_filter.run_image_referencer = licensing_integration.should_run_image_referencer()
-
         runner_filter.filtered_policy_ids = policy_metadata_integration.filtered_policy_ids
         logger.debug(f"Filtered list of policies: {runner_filter.filtered_policy_ids}")
 
         runner_filter.excluded_paths = runner_filter.excluded_paths + list(repo_config_integration.skip_paths)
-
-        runner_filter.set_suppressed_policies(suppressions_integration.get_policy_level_suppressions())
+        policy_level_suppression = suppressions_integration.get_policy_level_suppressions()
+        bc_cloned_checks = custom_policies_integration.bc_cloned_checks
+        runner_filter.bc_cloned_checks = bc_cloned_checks
+        custom_policies_integration.policy_level_suppression = policy_level_suppression
+        runner_filter.set_suppressed_policies(policy_level_suppression)
```

Comment:
looks like you have this block of code 3 times here, can you extract to a function?

Reply:
Not needed IMO (we won't reuse it anywhere else than main)